cmd/juju/commands/main_test.go

```diff
@@ -203,10 +203,16 @@ func (s *MainSuite) TestFirstRun2xFrom1xOnUbuntu(c *gc.C) {
 
 	c.Check(code, gc.Equals, 0)
 	c.Check(string(stderr), gc.Equals, fmt.Sprintf(`
-    Welcome to Juju %s. If you meant to use Juju 1.25.0 you can continue using it
-    with the command juju-1 e.g. 'juju-1 switch'.
+Welcome to Juju %s. 
     See https://jujucharms.com/docs/stable/introducing-2 for more details.
 
+If you meant to use Juju 1.25.0, run 'juju' commands as 'juju-1'. For example, 'juju switch' as 'juju-1 switch'.
+    Note: 
+    'juju-1' command comes from juju-1-default package.
+    On trusty, use 'update-alternatives'.
+    On xenial/yakkety, use juju-1-default package install.
+    After the installation, you should have /usr/bin/juju-1 directory to use 'juju-1' command. 
```

Comment:
Removed word "directory" altogether. Still reads fine and I'd like to avoid confusion that may come with calling symlink binary or the other way around...